docs/community/README.md

```diff
@@ -6,19 +6,19 @@ To interact with the Observatorium community use any of the following channels:
 
 ## Issues, Bugs, Ideas, Questions
 
-For any asynchronous communication, please fill [github.com/observatorium/observatorium](https://github.com/observatorium/observatorium) GitHub Issue or start Discussion thread.
+Before starting a new thread, please make sure to search for any previous relevant conversations.
 
-Before doing that, make sure to search for any previous relevant conversations.
+For any asynchronous communication, please file an [github.com/observatorium/observatorium](https://github.com/observatorium/observatorium) GitHub Issue or start a GitHub Discussion.
```

Comment:
```suggestion
For any asynchronous communication, please file a [github.com/observatorium/observatorium](https://github.com/observatorium/observatorium) GitHub Issue or start a GitHub Discussion.
```

Reply:
I made it `an` because I thought it would look read better with the URL, but maybe the problem is that we have a URL in the middle of the sentence in the first place. Should we change that?